test/python/classical_function_compiler/test_parse.py

```diff
@@ -11,38 +11,46 @@
 # that they have been altered from the originals.
 
 """Tests the classicalfunction parser."""
+import unittest
 
 from qiskit.circuit.classicalfunction import ClassicalFunctionParseError
 from qiskit.circuit.classicalfunction import classical_function as compile_classical_function
+from qiskit.circuit.classicalfunction.classicalfunction import HAS_TWEEDLEDUM
+
 from qiskit.test import QiskitTestCase
 from . import bad_examples as examples
 
 
 class TestParseFail(QiskitTestCase):
     """Tests bad_examples with the classicalfunction parser."""
 
+    @unittest.skipUnless(HAS_TWEEDLEDUM, 'tweedledum not available')
```

Comment:
This doesn't need to exist, you shouldn't need to raise a skip exception on an assert method